torch/_inductor/ir.py

```diff
@@ -2259,10 +2259,14 @@ def unflatten_args(new_tensor_args, new_non_tensor_args):
                     new_args.append(next(it_non_tensors))
             return pytree.tree_unflatten(new_args, args_spec)
 
-        tensor_args = [
-            cls.require_contiguous(cls.realize_input(x)) for x in tensor_args
-        ]
-
+        if kernel == torch.ops.aten.convolution:
+            tensor_args = [
+                cls.require_stride1(cls.realize_input(x)) for x in tensor_args
+            ]
+        else:
```

Comment:
Is this change covered by an existing test?

Reply:
Yes, it is covered.

Reply:
As we discussed offline, we'd better check why the issue was not exposed when kernel size is 1 as well.